deploy/ray-operator/config/samples/ray_v1_raycluster.bk.yaml

```diff
@@ -0,0 +1,294 @@
+apiVersion: ray.io/v1
+kind: RayCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: raycluster-sample
+spec:
+  clusterName: raycluster-sample
+  images:
+    defaultImage: "reg.docker.alibaba-inc.com/volvo_ci_arcx/ray_package:a59ce20ec8b9876a67df84ca45dc456481bc2d3a"
+    clusterApiServerImage: "reg.docker.alibaba-inc.com/zeromem/clusterapiserver:20191008"
+  imagePullPolicy: "Always"
+
+  # optional `extensions`, and all sub-options are optional.
+  extensions:
```

Comment:
How about split extensions into "headGroup" and "workerGroups"?

Reply:
this file deleted. 

- for now we support head/worker group.But we can have multiple worker groups to support heterogeneous
- didn't get meaning of workerGroups, a group of group workers?

Reply:
Yes,workerGroups is a group of group workers

Reply:
- Technically,  it's ok to user workerGroups.
- The autoscaler yaml will make the same change.  

Below is part of autoscaler yaml, workerGroups will make this file change
```yaml
worker_nodes:
    apiVersion: v1
    kind: Pod
    ...
```

There will be two designs showing below.

1. One kind, flat 
```yaml
worker_nodes:
    apiVersion: v1
    kind: Pod
    ...

worker_nodes_1:
    apiVersion: v1
    kind: Pod
    ...

worker_nodes_2:
    apiVersion: v1
    kind: Pod
    ...
```

2.The other one, gathering
```yaml
worker_nodes:
   [
    apiVersion: v1
    kind: Pod
    ...
   ],
  [
    apiVersion: v1
    kind: Pod
    ...
   ],
    ...
```